plugins/org.jkiss.dbeaver.ext.postgresql/plugin.xml

```diff
@@ -328,6 +328,7 @@
                     <parameter name="serverType" value="postgresql"/>
                     <property name="loginTimeout" value="20"/>
                     <property name="connectTimeout" value="20"/>
+                    <property name="escapeSyntaxCallMode" value="callIfNoReturn"/>
```

Comment:
So... is debugging available only for Postgres? What about Greenplum? Cockroach? EDB? Redshift? Yellowbrick? etc.

Reply:
I believe so. To use the debugger, one must install the native extension for the database.

Reply:
But EDB (as an example) is a kind of clone of Postgres. And supports extensions. Are you sure?

Reply:
I guess it's indeed compatible with EDB (judging by the "about" section https://github.com/EnterpriseDB/pldebugger).